content/6_Advanced/Treaps.problems.json

```
@@ -10,7 +10,23 @@
       "isStarred": false,
       "tags": [],
       "solutionMetadata": {
-        "kind": "none"
+        "kind": "in-module",
+        "moduleId": "treaps"
+      }
+    }
+  ],
+  "dsrars": [
+    {
+      "uniqueId": "ys-DynamicSequenceRangeAffineRangeSum",
+      "name": "Dynamic Sequence Range Affine Range Sum",
+      "url": "https://judge.yosupo.jp/problem/dynamic_sequence_range_affine_range_sum",
+      "source": "YS",
+      "difficulty": "Easy",
+      "isStarred": true,
+      "tags": ["treap"],
```

```suggestion
      "tags": ["Treap"],
```
just to stay consistent with other tags